service/provider/src/main/java/gov/va/vro/service/provider/mas/service/mapper/MasCollectionAnnotsResults.java

```diff
@@ -45,35 +46,43 @@ public AbdEvidence mapAnnotsToEvidence(MasCollectionAnnotation masCollectionAnno
       if (masDocument.getAnnotations() != null) {
         for (MasAnnotation masAnnotation : masDocument.getAnnotations()) {
           log.info(
-              ">>>> Annotation Tpe <<<<<< : {} ",
+              ">>>> Annotation Type <<<<<< : {} ",
               MasAnnotType.fromString(masAnnotation.getAnnotType().toLowerCase()));
           MasAnnotType AnnotationType =
               MasAnnotType.fromString(masAnnotation.getAnnotType().toLowerCase());
           switch (AnnotationType) {
             case MEDICATION -> {
               AbdMedication abdMedication = new AbdMedication();
+              abdMedication.setDataSource(DATA_SOURCE);
               abdMedication.setStatus(null);
               abdMedication.setNotes(null);
               abdMedication.setDescription(masAnnotation.getAnnotVal().toLowerCase());
               abdMedication.setRefills(-1);
               abdMedication.setAsthmaRelevant(null);
               abdMedication.setDuration(null);
               if (masAnnotation.getObservationDate() != null) {
-                abdMedication.setAuthoredOn(masAnnotation.getObservationDate().replaceAll("Z", ""));
+                abdMedication.setAuthoredOn(
+                    masAnnotation.getObservationDate().replaceAll("Z", "") + UTC_TM);
               } else {
-                abdMedication.setAuthoredOn("9999-12-31");
+                abdMedication.setAuthoredOn("9999-12-31" + UTC_TM);
               }
               abdMedication.setRoute(null);
               abdMedication.setAsthmaRelevant(isConditionAsthma);
               medications.add(abdMedication);
             }
             case CONDITION -> {
               AbdCondition abdCondition = new AbdCondition();
+              abdCondition.setDataSource(DATA_SOURCE);
               abdCondition.setCode(masAnnotation.getAnnotVal());
               abdCondition.setText(masAnnotation.getAcdPrefName());
               abdCondition.setStatus(null);
               abdCondition.setAbatementDate(null);
-              abdCondition.setOnsetDate(masAnnotation.getObservationDate());
+              if (masAnnotation.getObservationDate() != null) {
+                abdCondition.setOnsetDate(
```

Comment:
We should double check this mapping. I would think that observationDate more closely corresponds to Condition.recordedDate rather than Condition.onsetDate. It looks like the Lighthouse field extractor also uses onsetDate, however, the PDF design is supposed to display recordedDate. I can bring it up with the design people.

Reply:
It's interesting also that from Lighthouse, `recordedDate` has the same format as `observationDate` from MAS. (example: 'recordedDate': '2013-05-15') and `onsetDate` has the extended time format (example: 'onsetDateTime': '2013-05-15T18:16:07Z') So if we map to `observationDate` to `recordedDate` we will not have to manipulate the data as much.